enter.sh

```diff
@@ -6,4 +6,5 @@ docker run -i -t --rm \
 	--mount type=bind,source=`pwd`/microph,target=/work/microph \
 	--mount type=bind,source=`pwd`/shalconv,target=/work/shalconv \
 	--mount type=bind,source=`pwd`/noah_lsm,target=/work/noah_lsm \
+	--mount type=bind,source=`pwd`/runfile,target=/work/runfile \
```

Comment:
Great catch. I used a dev container which mounts the whole directory, thanks for adding this!